src/data_handler.py

```diff
@@ -0,0 +1,167 @@
+#import BoardInterface
+
+from enum import Enum
+
+class Sensor():
+
+    def __init__(self, sensorID, boardID, name):
+        self.sensorID = sensorID
+        self.boardID = boardID
+        self._lastValue = 0.0
+        self.name = name
+
+    def getSensorValue():
+        return _lastValue
+
+    def setSensorValue(sensorVal):
+        _lastValue = sensorVal
+
+    def getName():
+        return name
+
+    def _rawToRealValue(rawValue):
+        pass
+
+    def getSensorID():
+        return sensorID
+
+    def getBoardID():
+        return boardID
+
+class PressureSensor(Sensor):
+
+    def _rawToRealValue(rawValue):          #take in as an int value between 0 and 0xFFF
+        return rawValue * (1500 / 4095)         #Maps from (0x000,0xFFF) to (0,1500) psi
+
+class TempSensor(Sensor):
+
+    def _rawToRealValue(rawValue):
+        integer = int(rawValue >> 5)
+        decimal = float(int(rawValue % (2**6))) / 32.0
```

Comment:
Please comment what you're doing here. Also, shouldn't this be 2^5?

Reply:
oops my bad u rite